src/annotator/config/test/settings-test.js

```diff
@@ -22,245 +27,42 @@ describe('annotator/config/settingsFrom', () => {
-          }, 'application/annotator+html (rel="notebook") link has no href');
-        });
-      });
-
-      context("when there's no annotator+html notebook link", () => {
-        it('throws an error', () => {
-          assert.throws(() => {
-            settingsFrom(window).notebookAppUrl; // eslint-disable-line no-unused-expressions
-          }, 'No application/annotator+html (rel="notebook") link in the document');
-        });
-      });
     });
+  });
 
-    describe('#sidebarAppUrl', () => {
-      context("when there's an application/annotator+html sidebar link", () => {
-        let link;
-
-        beforeEach('add an application/annotator+html sidebar <link>', () => {
-          link = appendLink('http://example.com/app.html', 'sidebar');
-        });
-
-        afterEach('tidy up the sidebar link', () => {
-          link.remove();
-        });
-
-        it('returns the href from the sidebar link', () => {
-          assert.equal(
-            settingsFrom(window).sidebarAppUrl,
-            'http://example.com/app.html'
-          );
-        });
-      });
-
-      context('when there are multiple annotator+html sidebar links', () => {
-        let link1;
-        let link2;
-
-        beforeEach('add two sidebar links to the document', () => {
-          link1 = appendLink('http://example.com/app1', 'sidebar');
-          link2 = appendLink('http://example.com/app2', 'sidebar');
-        });
-
-        afterEach('tidy up the sidebar links', () => {
-          link1.remove();
-          link2.remove();
-        });
-
-        it('returns the href from the first one', () => {
-          assert.equal(
-            settingsFrom(window).sidebarAppUrl,
-            'http://example.com/app1'
-          );
-        });
-      });
-
-      context('when the annotator+html sidebar link has no href', () => {
-        let link;
-
-        beforeEach(
-          'add an application/annotator+html sidebar <link> with no href',
-          () => {
-            link = appendLink(null, 'sidebar');
-          }
-        );
-
-        afterEach('tidy up the sidebar link', () => {
-          link.remove();
-        });
-
-        it('throws an error', () => {
-          assert.throws(() => {
-            settingsFrom(window).sidebarAppUrl; // eslint-disable-line no-unused-expressions
-          }, 'application/annotator+html (rel="sidebar") link has no href');
-        });
-      });
-
-      context("when there's no annotator+html sidebar link", () => {
-        it('throws an error', () => {
-          assert.throws(() => {
-            settingsFrom(window).sidebarAppUrl; // eslint-disable-line no-unused-expressions
-          }, 'No application/annotator+html (rel="sidebar") link in the document');
-        });
-      });
+  describe('#sidebarAppUrl', () => {
+    it('calls urlFromLinkTag with appropriate params', () => {
+      assert.exists(settingsFrom(window).sidebarAppUrl);
```

Comment:
Same situation, I strongly feel we should not have redundant tests. If we start to test things in a deeper mannar, than our unit tests look less like unit tests and more like end to end tests.  The reason I called .exists is so lint would pass. I really don't need need any tests here, but since I have to have one, it could simply test the value, however this makes it somewhat redundant.